common/test/acceptance/pages/studio/library.py

```diff
@@ -168,6 +169,7 @@ def count(self, count):
         Sets value of children count input
         """
         count_text = self.get_setting_element(self.COUNT_LABEL)
+        count_text.send_keys(Keys.BACK_SPACE)
```

Comment:
Ah, I understand. However because it is a method in the page object, our goal should be that it work universally so that other people can use the method in future tests without understanding this nuance.
As long as you are touching (and improving!) the code you should change it to select all then delete (and remove the .clear() on the next line.)